### ISSUE TITLE
First list is an implicit article update list showing a changelog

### ISSUE THREAD

**Issue by novoid** [novoid/lazyblorg#20]
Hi,
So far, I was manually writing paragraphs like following to update to [pages like this](https://karl-voit.at/managing-digital-photographs/):

```
Update 2014-05-14: added real world example

Update 2015-03-16: filtering photographs according to their GPS coordinates

Update 2016-08-29: replaced outdated ~show-sel.sh~ method with new ~filetags --filter~ method

Update 2017-08-28: Email comment on geeqie video thumbnails

[begin of the article content]
```

What about making an implicit changelog list?

Any simple itemize list at the begin of an blog article (which was not possible so far because of #10) is implicitly interpreted (and formatted) as a changelog list with special CSS markup.

For example:

```
- Update 2014-05-14: added real world example
- Update 2015-03-16: filtering photographs according to their GPS coordinates
- Update 2016-08-29: replaced outdated ~show-sel.sh~ method with new ~filetags --filter~ method
- Update 2017-08-28: Email comment on geeqie video thumbnails

[begin of the article content]
```
... may be transformed to:
```
- Article changelog:
  - 2014-05-14: added real world example
  - 2015-03-16: filtering photographs according to their GPS coordinates
  - 2016-08-29: replaced outdated ~show-sel.sh~ method with new ~filetags --filter~ method
  - 2017-08-28: Email comment on geeqie video thumbnails

[begin of the article content]
```
... or similar.

I am not sure whether or not the actual words should be changed. Maybe just adapt the CSS markup (smaller font, different background, ...).

The goal should be that the reader is clearly able to see that this is meta-data and that the actual content is started below.

**Comment by xeruf** [comment]
how about going further and extracting article changelogs from something like git commit history or via the metadata generation, which already seems to track article changes somewhat?

**Comment by novoid** [comment]
> how about going further and extracting article changelogs from something like git commit history or via the metadata generation, which already seems to track article changes somewhat?

Yes, for somebody who is keeping all lazyblorg-OD-files in a git repo and manually typing commit messages, that would maybe a good idea. That could be done via an elisp function within Emacs.

I don't commit by hand and therefore my commit messages do not contain any helpful information at all.

> Oh and again, I would deem reverse chronological ordering more sensible ;)

:-)

This could be made into a config option I guess. Should not be that hard. I've created #41 to track this idea. Maybe you want to implement it? It should be rather simple and I'd answer questions in order to help you if you want.

As always: it's a "runs on my machine" project that was never designed to be used by many peers. Therefore, I don't want to implement any functionality I'm not using myself. Maybe I should emphasize that in the README ...

Anyway, the order of the update list is totally up to you.
